is_core/utils/compatibility.py

```diff
@@ -12,3 +12,10 @@ def __call__(self, request):
         if hasattr(self, 'process_response'):
             response = self.process_response(request, response)
         return response
+
+
+def get_last_parent_pk_field_name(obj):
```

Comment:
This function is from [django-pyston](https://github.com/vichi99/django-pyston/blob/9b5a2859e8d611522e141f1b0d8bc563adbbd1ed/pyston/utils/compatibility.py#L107).  I find it more convenient to use it than to use only `pk`